working_time_tracker/create_superuser.py

```diff
@@ -0,0 +1,15 @@
+import os
+import django
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "working_time_tracker.settings")
+django.setup()
+
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+username = os.getenv("DJANGO_SUPERUSER_USERNAME")
+password = os.getenv("DJANGO_SUPERUSER_PASSWORD")
+
+if not User.objects.filter(username=username).exists():
```

Comment:
Good job
Checking for the user is a nice solution to not overpopulate the database